src/main/drivers/dshot_bitbang_decode.h

```diff
@@ -22,7 +22,7 @@
 
 #if defined(USE_DSHOT) && defined(USE_DSHOT_TELEMETRY)
 
-#if defined(STM32F4) || defined(AT32F4)
+#ifdef USE_DSHOT_BITBAND
 uint32_t decode_bb_bitband( uint16_t buffer[], uint32_t count, uint32_t bit);
```

Comment:
It's BITBAND or BITBANG?

Reply:
Band, although I’ve not investigated quite what that is!